docker-compose.yml

```diff
@@ -285,7 +285,7 @@ services:
       - OTEL_COLLECTOR_HOST
       - FLAGD_HOST
       - FLAGD_PORT
-      - BUGSNAG_API_KEY
+      - FRONTEND_BUGSNAG_API_KEY
```

Comment:
Based on how the Terraform configuration looks, it seems that we need to keep the `BUGSNAG_API_KEY` in the app and add the mapping in here, so what should work is:
```
- BUGSNAG_API_KEY=${FRONTEND_BUGSNAG_API_KEY}
```
So in the `.env` file we will have `FRONTEND_BUGSNAG_API_KEY` and in the code in the app `BUGSNAG_API_KEY`.

Please let me know if it works.